ultraplot/axes/base.py

```diff
@@ -2273,6 +2339,8 @@ def _legend_tuple(*objs):  # noqa: E306
             else:
                 hs = hs[0] if len(hs) == 1 else hs  # unfurl for better error messages
                 label = label if label is not None else labs[0] if labs else "_no_label"
+                if label and str(label).startswith("_"):
+                    continue
```

Comment:
@cvanelteren This change is not a simple refactor for clarity. The continue statement means that labels that start with `_` are never added to the list. What happens if a user passes a label like `_my_custom_label`?

Reply:
This is how it was  before:
https://github.com/Ultraplot/ultraplot/blob/eb7fef59bfa1fc43a4cba61eb55b10f9977b2178/ultraplot/axes/base.py#L2214

Reply:
ah oops listing the wrong thing. I will revert this

Reply:
For context, I think this was there due to the internal maps having weird `_` starting labels. But can't recall exactly why I wrote this. It has been a little while :)